src/components/forms/DateInput.jsx

```diff
@@ -222,6 +223,7 @@ const DateInput = ({
 
   const handleInputChange = ({ target: { value } }) => {
     setInputValue(value);
+    onInputChange();
```

Comment:
should we provide either the value, or the event? e.g. `onInputChange(value)`.

If we would pick one of them, I'd choose the event just to be consistent with the normal `onChange` but not too opinionated here.